src/dict.c

```diff
@@ -534,6 +534,54 @@ void *dictFetchValue(dict *d, const void *key) {
     return he ? dictGetVal(he) : NULL;
 }
 
+/* Find an element from the table, also get the plink of the entry. The entry
```

Comment:
What does plink mean in this context stand for? It's a pointer to some link?

Reply:
plink is the reference to the pointer that points to the entry to be removed.
it doesn't really matter for the caller, it's just a variable they need to pass to the other function.
what's missing here in your mind? just an explanation of what `plink` means?

Reply:
Yeah, I think I was just confused reading it.